frontend/scripts/openapi.yaml

```diff
@@ -10567,8 +10567,13 @@ components:
           type: integer
           readOnly: true
           description: Total number of certified contributors who rated the video
+        duration:
+          type: integer
+          readOnly: true
+          description: Duration of the video
```

Comment:
This file does not seem to be up to date with the API. This field `duration` is supposed to be "nullable".